### DISCUSSION TITLE
刚接触Ethersjs 为什么一直连接超时呀

### DISCUSSION THREAD

**Discussion by TsumikiQAQ** [AmazingAng/WTF-Solidity#463] [Q&A] [answered]
弄了三天了 挂梯子 卸载重装 改环境 重装系统都试过了 ┭┮﹏┭┮
![代码](https://user-images.githubusercontent.com/116857998/228404071-30962070-e106-4fb2-be7c-a914a5f43d2c.png)
![报错](https://user-images.githubusercontent.com/116857998/228404080-9b3f44d9-f5c1-42f1-91db-62ed2ab98b89.png)

**Comment by riskers** [comment, marked as answer]
getBalance 是 await 方法

```js
const ethers = require('ethers')
const provider = new ethers.getDefaultProvider()
;(async () => {
  const balance = await provider.getBalance(`vitalik.eth`)
  console.log(balance)
})()
```
